src/utilities/apiCalls.ts

```diff
@@ -1,14 +1,13 @@
-const apiKey = "";
+const api = "http://localhost:3000/productList";
```

Comment:
put server address to .env file and import variable here
https://cli.vuejs.org/guide/mode-and-env.html#environment-variables